core/src/edu/asu/c3simulator/screens/CouncilPanel.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Navigation Panel for council screens
+ * 
+ * @author Alyahya, Mohammed
```

Comment:
We need to verify that this class isn't registered in any other branches or commits.